### PR TITLE
Reverse dns docs reorg

### PR DESCRIPTION
**Pull request by ksigler7**
- [x] Taken all valuable information from existing pages and added it to the new category
- [x] Deleted old pages
- [x] Deleted old images from images folder
- [x] Added remaining images to the img folder
- [x] Screenshots updated
- [x] all images have an alt tag
- [x] all images are named according to style guide
- [x] Updated the glossary
- [x] Added redirects from deleted page to new content
- [x] Head data should have SEO title, keywords, and description
- [x] Content is in Jobs-to-be-done format
- [x] Update all in-content links and links to deleted pages
- [x] Added additional resources (aim for a link to the glossary, the blog and other jobs-to-be-done)
- [x] Added a TOC
- [x] Added a talk to Dewey/Olivia section
- [x] title optimized for google search snippits
- [x] content optimized for google search snippits
- [x] keyword optimization for each category
- [x] addressed/closed any related issues in the backlog
- [x] pages are have the correct weight
- [x] content has been run through grammarly
- [x] properly trademarked names for any product names outside of SendGrid

### CODE REVIEW

**Comment by ksigler7** [comment]
@Whatthefoxsays ready for your review!!

File: source/Glossary/reverse_dns.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ navigation:
   show: false
 ---
 
-Setting up reverse DNS on IP addresses allows mailbox providers to verify the sender when they do a reverse DNS lookup upon receipt of the emails you send. This differs from domain authentication in that, while you should have your IP addresses in your SPF records, setting up reverse DNS on the IP addresses does not require a change to SPF or DKIM records.
+Setting up reverse DNS on IP addresses allows mailbox providers to verify the sender when they do a reverse DNS lookup upon receipt of the emails you send. When you update your DNS provider with a DNS record provided by SendGrid, and then send mail over your IP, the recipient's email service provider performs a reverse DNS lookup (rDNS) using an A Record (address record). An A record maps your domain to your IP address. When a mailbox provider looks up your A Record, they see your SendGrid IP address. When they look at your IP address, they see the rDNS that match your A Record. This circular checking proves your SendGrid IP association with your domain and your domain association with your SendGrid IP.
```
Comment (Whatthefoxsays) [inline source/Glossary/reverse_dns.md]: An A Record maps your domain to your IP address. (Record wasn't capitalized)

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md [hunk 2]
```diff
@@ -8,128 +8,114 @@ seo:
...
 
-{% info %}
-Only the parent account can whitelabel IP addresses. The parent account can then assign the IPs to the subusers accordingly.
-{% endinfo %}
+{% anchor h2 %}
+What is reverse DNS?
+{% endanchor %}
 
-An individual IP address can only be whitelabeled to a single domain, but a domain can have many IPs whitelabeled to it. This allows for you to use IP pooling for your email sending.
+Setting up reverse DNS on IP addresses allows mailbox providers to verify the sender when they do a reverse DNS lookup upon receipt of the emails you send. When you update your DNS provider with a DNS record provided by SendGrid, and then send mail over your IP, the recipient's email service provider performs a reverse DNS lookup (rDNS) using an A Record (address record). An A record maps your domain to your IP address. When a mailbox provider looks up your A Record, they see your SendGrid IP address. When they look at your IP address, they see the rDNS that match your A Record. This circular checking proves your SendGrid IP association with your domain and your domain association with your SendGrid IP.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md]: An A Record maps your domain to your IP address. (capitalize Record)

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md [hunk 3]
```diff
@@ -8,128 +8,114 @@ seo:
...
-{% info %}
-We suggest that you use the same subdomain as your domain whitelabel.
-{% endinfo %}
-
-We will then prepend a second level subdomain similar to ```o1``` incrementing the numeric value of this string as you add IP whitelabels to this subdomain.
+DNS stands for Domain Name System. This is a naming system for domains on the internet. When SendGrid refers to your DNS, we are talking about your domain name that you want to send emails from, or that you want to link images from. When we talk about your DNS provider, we are talking about the service that hosts your domain name. For example, GoDaddy, Rackspace, or Cloudflare. For more information about DNS, see our [DNS glossary page]({{root_url}}/Glossary/dns.html).
 
 {% anchor h3 %}
-Domain
+A record
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md]: Are we capitalizing Record? I've seen it multiple ways. Pick your favorite. :)

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md [hunk 4]
```diff
@@ -8,128 +8,114 @@ seo:
...
+1. In the reverse DNS section, click **Get Started**.
+1. Next, select the IP to set up reverse DNS.
+1. Add a subdomain. For more information about picking a subdomain, see [Subdomains](#-Subdomain).
+1. Select or enter a domain you want to associate with the IP. Make sure that you only enter the name of your root domain. Do not include `www` or `http://www` in this field! Your domain needs to match the domain of your from address on the emails you are sending out. For example, if I am setting up the domain `example.sendgrid.com`, I would set my domain to be `sendgrid.com`. Click **Save** and then **Confirm**. An individual IP address can only have reverse DNS set up for a single domain, but a domain can have reverse DNS set up for multiple IPs. This allows for you to use IP pooling for your email sending.
+1. Next, you need to add all of the A records on this screen to your DNS host. This process varies depending on your DNS host. For videos on how to add your A records to some popular DNS service providers, check out these [videos](https://sendgrid.com/docs/User_Guide/Settings/Whitelabel/providers.html).
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md]: A records/A Records

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md [hunk 5]
```diff
@@ -8,128 +8,114 @@ seo:
...
 
 {% anchor h2 %}
 What do I do if I have more than 10 IPs
 {% endanchor %}
 
-There is a character limit in SPF records that means that if you have more than 10 IP addresses, they will not fit in the record. When this is true, we will provide you with the generic SendGrid SPF record which includes all IPs at SendGrid, not just yours. If you would like to secure your SPF record to only include your IPs, you can chain multiple SPF records together manually. For more information see Open SPF’s website.
+There is a character limit in SPF records that means that if you have more than 10 IP addresses, they will not fit in the record. When this is true, we will provide you with the generic SendGrid SPF record which includes all IPs at SendGrid, not just yours. If you would like to secure your SPF record only to include your IPs, you can chain multiple SPF records together manually. For more information see Open SPF’s website.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_reverse_dns.md]: Do you need to spell out SPF in the first instance?

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**